src/Sharpbrake.Client/NoticeBuilder.cs

```diff
@@ -3,16 +3,19 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Runtime.CompilerServices;
 using System.Runtime.Serialization.Json;
 using System.Text;
 using Sharpbrake.Client.Model;
 
+[assembly: InternalsVisibleTo("Sharpbrake.Client.Tests")]
```

Comment:
So this class is not accessible to external libraries, but you can access it in tests. That's pretty cool! I haven't seen anything like this for other languages (but probably something like this exists elsewhere, too 😀)